runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -1891,6 +1902,32 @@ class INTERPRETER_CLASS
 				}
 				break;
 			}
+#if defined(J9VM_OPT_MICROJIT)
+			if (mjitExtra) {
+				if (J9_ARE_NO_BITS_SET(mjitExtra, J9_STARTPC_NOT_TRANSLATED)) {
+					/* Already compiled with MJIT */
+					runMethodCompiled = true;
+				} else if (mjitExtra < 0) {
+					// This method should not be translated by MJIT (already failed), but must keep counting
```

Comment:
Use `/* */` comments as preferred by vm code
```suggestion
					/* This method should not be translated by MJIT (already failed), but must keep counting */
```

Reply:
Similar concern about this never being reached due to a `break` statement above